src/app/clusters/illuminance-measurement-server/illuminance-measurement-cluster.h

```diff
@@ -0,0 +1,66 @@
+/*
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app/server-cluster/DefaultServerCluster.h>
+#include <app/server-cluster/OptionalAttributeSet.h>
+#include <platform/DeviceInfoProvider.h>
+
+namespace chip::app::Clusters {
+
+class IlluminanceMeasurementCluster : public DefaultServerCluster
+{
+public:
+    // NOTE: this set is smaller than the full optional attributes supported by time synchronization
+    //       as other attributes are controlled by feature flags
+    using OptionalAttributeSet = app::OptionalAttributeSet<IlluminanceMeasurement::Attributes::LightSensorType::Id,
+                                                           IlluminanceMeasurement::Attributes::Tolerance::Id>;
+
+    struct StartupConfiguration
+    {
+        IlluminanceMeasurement::LightSensorTypeEnum lightSensorType{ IlluminanceMeasurement::LightSensorTypeEnum::kNone };
+        IlluminanceMeasurement::Attributes::Tolerance::TypeInfo::Type tolerance{};
```

Comment:
Thinking that config should also contain the min/max of values: those are read-only values set by the app it seems, and the rule is `min <= measured <= max` (or measured is 0 if it cannot be measured ... which is odd for nullable values ... but ok)